src/json.erl

```diff
@@ -1376,7 +1376,10 @@ validate_json([{K, V}  | T], JSON, State = #state{plain_string = Plain}) ->
 %% Numeric
 validate_prop(multipleOf, _, JSON, _) when not is_number(JSON) -> true;
 validate_prop(multipleOf, N, JSON, _) when is_number(N), N > 0 ->
-    0.0 = (JSON / N - trunc(JSON / N)) * N;
```

Comment:
Simpler to just change to  0.0 == (JSON / N - trunc(JSON / N)) * N;

Reply:
Yes, but didn't want to change that it was returning 0.0 in this row. And using `==` would return a true. I do a change.